intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/verifiers/ProductReleaseVersionVerifier.kt

```diff
@@ -0,0 +1,90 @@
+package com.jetbrains.plugin.structure.intellij.verifiers
+
+import com.jetbrains.plugin.structure.base.problems.NotNumber
+import com.jetbrains.plugin.structure.base.problems.PropertyNotSpecified
+import com.jetbrains.plugin.structure.intellij.beans.PluginBean
+import com.jetbrains.plugin.structure.intellij.problems.ReleaseVersionAndPluginVersionMismatch
+import com.jetbrains.plugin.structure.intellij.problems.ReleaseVersionWrongFormat
+import com.jetbrains.plugin.structure.intellij.verifiers.ProductReleaseVersionVerifier.VerificationResult.Invalid
+import com.jetbrains.plugin.structure.intellij.version.ProductReleaseVersion
+
+class ProductReleaseVersionVerifier {
+  fun verify(plugin: PluginBean, descriptorPath: String, problemRegistrar: ProblemRegistrar): VerificationResult {
+    if (plugin.productDescriptor == null) return VerificationResult.NotApplicable
+
+    val releaseVersionValue = plugin.productDescriptor?.releaseVersion
+    if (releaseVersionValue.isNullOrEmpty()) {
+      return Invalid("Attribute '$RELEASE_VERSION_ATTRIBUTE_NAME' is missing").also {
+        problemRegistrar.registerProblem(PropertyNotSpecified(RELEASE_VERSION_ATTRIBUTE_NAME, descriptorPath))
+      }
+    }
+
+    return try {
+      ProductReleaseVersion.parse(releaseVersionValue).run {
+        if (isSingleDigit) {
+          Invalid("Attribute '$RELEASE_VERSION_ATTRIBUTE_NAME' must have two or more digits: '$releaseVersionValue'").also {
```

Comment:
Will it be possible to remap this to `warning` for old plugins, which still have one digit in the release-version?